otio_cookelensmetadata/adapter.py

```diff
@@ -100,9 +105,18 @@ def _decode_timecode(
     return tc_string
 
 
+def _normalize_rate( in_rate: float 
+) -> float:
+    """
+    Normalizes a frame rate float to one of the standard rates
+    """
+    return round(in_rate * 1001)/1001
```

Comment:
Love this approach, I haven't seen it before!

Reply:
I just made it up; but I think it's correct!  Should work for any standard frame rate.